modules/stm/test/test_stm.py

```diff
@@ -7,13 +7,19 @@
 
 from osdtestlib.debug_interconnect import NocDriver, RegAccess, DiPacket
 from osdtestlib.eval_utility import EvalUtility
+from osdtestlib.eval_utility import Standard
+from osdtestlib.soc_interface import TraceGenerator
+
+from enum import Enum
 
 @cocotb.test()
 def test_stm(dut):
     """
     Try accessing the design
     """
 
+    MODULE_ID = 1
```

Comment:
Please rename this "MODULE_DI_ADDRESS" ("DI address" is the debug interconnect address, the name we try to use consistently. The old "id" naming should go some time soon.)